airbyte-db/lib/src/main/resources/jobs_database/Attempts.yaml

```diff
@@ -11,7 +11,7 @@ required:
   - status
   - created_at
   - updated_at
-additionalProperties: false
+additionalProperties: true
```

Comment:
Make the attempts schema forward compatible before the file-based migration system is removed.